src/items/equipment/equipment.js

```diff
@@ -82,7 +82,11 @@ export default class Equipment extends Item {
                     .playSfx(this.equipSound, false);
             } else {
                 console.log("Not high enough level to equip that.");
-                let skillText = getRequiredCombatSkill(this.skill);
+
+                const skillText =
+                    this.skill == EQUIPMENT.WEAPON_TYPES.MELEE
+                        ? "attack"
+                        : getRequiredCombatSkill(this.skill);
```

Comment:
Since we pass in `this.skill` into the method we can move this ternary logic there

Reply:
The ternary logic is different and not used for each usage of the method. It's a bit awkward and if it gets more complex in the future might look into refactoring the flow